docs/collections/redis-mode/create-redis-mode.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Create a Redis Mode Collection
+sidebar_position: 20
+---
+
+DINO - I copy/pasted this from the Document Store docs. I updated the Web Console tab, can you update the others?
```

Comment:
```suggestion

```

Not sure if this note is still needed, but it's not a best practice to include editorial notes within the text.  Best to write them down separately and include them in the GitHub review as comments.